### ISSUE TITLE
apiVersion: v2 not being accepted

### ISSUE THREAD

**Issue by half2me** [helm/chart-releaser-action#7]
I have a chart which requires Helm3, but when I try to use the:
`apiVersion: v2` in my `Chart.yaml` it fails with:
```
Error: apiVersion 'v2' is not valid. The value must be "v1"
```

**Comment by gtaylor** [comment]
Is this coming from the action or the upstream chart-releaser?

**Comment by unguiculus** [comment]
That's from chart-releaser. Helm 3 support is on its way: https://github.com/helm/chart-releaser/pull/47

**Comment by smark88** [comment]
> That's from chart-releaser. Helm 3 support is on its way: [helm/chart-releaser#47](https://github.com/helm/chart-releaser/pull/47)

This was merged, yet I'm stilling the same issue from v0.2.3 maybe they haven't released yet.

**Comment by unguiculus** [comment]
Fixed with latest release.
